Parse/src/test/java/com/parse/ParseACLTest.java

```diff
@@ -202,17 +205,18 @@ public void testResolveUserWithNewUser() throws Exception {
   public void testResolveUserWithUnresolvedUser() throws Exception {
     ParseACL acl = new ParseACL();
     ParseUser unresolvedUser = new ParseUser();
-    unresolvedUser.isLazy = true;
+    setLazy(unresolvedUser);
     // This will set the unresolvedUser in acl
     acl.setReadAccess(unresolvedUser, true);
     acl.setWriteAccess(unresolvedUser, true);
 
     unresolvedUser.setObjectId("test");
     acl.resolveUser(unresolvedUser);
+    unresolvedUser.setObjectId("test");
 
     assertNull(acl.getUnresolvedUser());
-    assertFalse(acl.getReadAccess(unresolvedUser));
-    assertFalse(acl.getWriteAccess(unresolvedUser));
+    assertTrue(acl.getReadAccess(unresolvedUser));
+    assertTrue(acl.getWriteAccess(unresolvedUser));
```

Comment:
Just to clarify this change, it looks like we were testing an illegal scenario originally, since there's no way for a `ParseUser` to be lazy and have an objectId, right?